### PR TITLE
Add JWT package changes to the release note

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

**Comment by sm1990** [comment]
LGTM

**Review by NipunaRanasinghe — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -95,3 +95,48 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 ### Bug fixes
 
 ## Backward-incompatible changes
+
+### Ballerina library changes
+
+#### `jwt` package
+
+- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
```
[the remark this answers is not in the document]
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:103]: Can we rephrase the last sentence?

E.g.,

With these additions, union types (or whatever that is not supported now) will be disallowed.

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -95,3 +95,48 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+    ```ballerina
+    // previous `jwt:IssuerSignatureConfig` record
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:105]: Capitalize the first letter and add a period at the end.

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:103]: Fixed with [89db88c9172a7a8cfc4671b2ef083645bbb74c7b](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/89db88c9172a7a8cfc4671b2ef083645bbb74c7b)

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -95,3 +95,48 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 ### Bug fixes
 
 ## Backward-incompatible changes
+
+### Ballerina library changes
+
+#### `jwt` package
+
+- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+    ```ballerina
```
[the remark this answers is not in the document]
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:104]: Fixed with [89db88c9172a7a8cfc4671b2ef083645bbb74c7b](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/89db88c9172a7a8cfc4671b2ef083645bbb74c7b)

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 2 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:105]: Fixed with [89db88c9172a7a8cfc4671b2ef083645bbb74c7b](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/89db88c9172a7a8cfc4671b2ef083645bbb74c7b)

**Pull request merged**